lib/p2p/NodeList.ts

```diff
@@ -74,6 +74,24 @@ class NodeList extends EventEmitter {
     this.nodes.forEach(callback);
   }
 
+  public rank = (): { primary: NodeInstance[], secondary: NodeInstance[] } => {
+    const primary: NodeInstance[] = [];
+    const secondary: NodeInstance[] = [];
+
+    this.nodes.forEach((node) => {
+      if (node.consecutiveConnFailures < 200) {
```

Comment:
I think it would be nice to name the 200 and 1500 values as `const` variables, it can just be somewhere in the header of this file for now. Something like `PRIMARY_PEER_CONN_FAILURE_THRESHOLD` and `SECONDARY_PEER_CONN_FAILURE_THRESHOLD` perhaps?

Reply:
I was thinking it will get clunky if we’ll add more logic here. But will fix.

Reply:
Yeah I don't think it's more logic per se, just naming variables so it's easier to see what the 200 and 1500 are for (and also a little easier to change or use elsewhere in the future, I believe).